domain/dagconfig/params.go

```diff
@@ -45,6 +45,42 @@ var (
 	devnetPowMax = new(big.Int).Sub(new(big.Int).Lsh(bigOne, 239), bigOne)
 )
 
+const (
+
+	// MaxCoinbasePayloadLength is the maximum length in bites allowed for a block's coinbase's payload
+	maxCoinbasePayloadLength = 150
```

Comment:
I think we still want these in constants, and use the default Params out of constants